### PR TITLE
Feature/model label zip

### PR DESCRIPTION
**Pull request by Perryongwq**
# Description

> Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Zipping of Model(.keras) and Label (.txt) files in CMT 

## Fixes / issues

1. Create backend program for zipping of the model and label files
2. Create a new container for hyperparameters and modelzip/save components
3. Create a frontend program to enable the function of modelzip
4. Clean up the backend code and improve UI/UX on alert notification 

## Type of change

> Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

File: frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  const set_parameters = (e) => {
+    setDrop((prevZip) => ({
+      ...prevZip,
+      [e.target.name]: {
+        ...prevZip[e.target.name],
+        selected: e.target.value,
+      },
+    }));
+
+    setParameters({ ...parameters, [e.target.name]: e.target.value });
```
Comment (cljiahao) [inline frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js:23]: This will result in the model chose in model zip to be used to retrain the model if user click train after choosing file in model save.

Comment (cljiahao) [inline frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js]: This will result in the model chose in model zip to be used to retrain the model if user click train after choosing file in model save.

File: backend/src/apis/routes/CMT_zip_file.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+router = APIRouter()
+
+class ModelName(BaseModel):
+    modelname: str
+
+@router.post("/zip_model/")
+async def zip_model(model: ModelName):
+    model_base_path = os.path.join(dire.temp_path, model.modelname)
+
+    txt_file_path = f"{model_base_path}.txt"
```
Comment (cljiahao) [inline backend/src/apis/routes/CMT_zip_file.py:17]: Filter out the datetime from the model name to save as just item type.

**Review by cljiahao — CHANGES_REQUESTED** [review]

File: frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js (hunk 1 above)
Comment (Perryongwq) [reply frontend/src/pages/CMT/containers/MenuChildren/components/ModelSave.js]: The issue is already resolve in the commit [f83a839](https://github.com/cljiahao/csam_ai_training/pull/45/commits/f83a839f3f19b53e43a3b3120426d2e370a385d8)

File: backend/src/apis/routes/CMT_zip_file.py (hunk 2 above)
Comment (Perryongwq) [reply backend/src/apis/routes/CMT_zip_file.py:17]: The issue is already resolve in the commit [f83a839](https://github.com/cljiahao/csam_ai_training/pull/45/commits/f83a839f3f19b53e43a3b3120426d2e370a385d8)

**Review by cljiahao — APPROVED** [review]
